tools/packaging/guest-image/build_image.sh

```diff
@@ -59,15 +60,16 @@ build_image() {
 	info "Build image"
 	info "image os: $os_name"
 	info "image os version: $os_version"
-	make image \
+	make invalidate-rootfs image \
 		DISTRO="${os_name}" \
 		DEBUG="${DEBUG:-}" \
 		USE_DOCKER="1" \
-		IMG_OS_VERSION="${os_version}" \
+		OS_VERSION="${os_version}" \
 		ROOTFS_BUILD_DEST="${builddir}/rootfs-image" \
 		AGENT_TARBALL="${AGENT_TARBALL}" \
 		AGENT_POLICY="${AGENT_POLICY:-}" \
 		PULL_TYPE="${PULL_TYPE:-default}" \
+		ARTEFACT_VERSION="${artefact_version}" \
```

Comment:
Does this also pass the artifact version to the `invalidate-rootfs` target?

Reply:
Yep. Same variables set for all targets called together.
make runs them sequentially.